content/sensu-go/5.20/reference/entities.md

```diff
@@ -38,6 +38,17 @@ All [commercial features][9] are available for free in the packaged Sensu Go dis
 If your Sensu instance includes more than 100 entities, [contact us][10] to learn how to upgrade your installation and increase your limit.
 See [the announcement on our blog][11] for more information about our usage policy.
 
+Commercial licenses include an entity limit and an entity class limit:
```

Comment:
Entity limit and entity class limit are not required. If undefined, the limit does not exist. Multiple entity class limits can exist.
```suggestion
Commercial licenses may include an entity limit and entity class limits:
```